Compiler/NFFrontEnd/NFTypeCheck.mo

```diff
@@ -1448,8 +1448,8 @@ function matchTypes
   input Type expectedType;
   input output Expression expression;
   input Boolean allowUnknown = false;
-        output Type compatibleType;
-        output Boolean compatible = true;
+  output Type compatibleType;
```

Comment:
I indent outputs in this way on purpose when using input output, since it makes it easier to spot all the outputs (notice that these two outputs line up with 'expression' above). So please don't change this.